library-model/library-model-generator/build.gradle

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2017. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id "java-library"
+    id "nullaway.java-test-conventions"
+}
+
+dependencies {
+    testImplementation deps.test.junit4
+    testImplementation(deps.build.errorProneTestHelpers) {
+        exclude group: "junit", module: "junit"
+    }
+    testImplementation project(":nullaway")
+    testImplementation project(":library-model:library-model-generator-integration-test")
```

Comment:
This is a weird almost-circular dependence.  Instead we should remove the dependence and the code under `src/test/java` in this module should move to the `library-model-generator-integration-test` module.

Reply:
moving it to the existing `library-model-generator-integration-test` was causing issues in getting the stubx file from the jar so I thought it would be best to do it exactly the way it is done for `jar-infer` and create a separate `test-library-model-generator` which has the test provider and the test java files and the `library-model-generator-integration-test` module now only holds `LibraryModelIntegrationTest.java` so it should be cleaner. Does this work?